### DISCUSSION TITLE
Fire Models and Fuel code

### DISCUSSION THREAD

**Discussion by filippi** [wirc-sjsu/firebench#11] [General]
Hi, nicely built and very high code quality, thanks.
Do you want us to directly add fire models in /src/firebench/rod_models/surface_for_vegetation.py  ?
Cheers

**Comment by Aurel31** [comment]
Hi Batti, 

I'll be happy to help you fork and directly edit  /src/firebench/rod_models/surface_for_vegetation.py to add your rate of spread models to the package. 

Best
